src/EFCore.Cosmos/Extensions/CosmosPropertyExtensions.cs

```diff
@@ -15,15 +15,15 @@ namespace Microsoft.EntityFrameworkCore
     public static class CosmosPropertyExtensions
     {
         /// <summary>
-        ///     Returns the property name used when targeting Cosmos.
+        ///     Returns the json property name used when targeting Cosmos.
```

Comment:
```suggestion
        ///     Returns the property name that the property is mapped to when targeting Cosmos.
```
In other places as well